src/main/java/io/split/android/client/storage/FileStorage.java

```diff
@@ -128,4 +128,10 @@ public boolean rename(String currentId, String newId) {
         File newFile = new File(_dataFolder, newId);
         return oldFile.renameTo(newFile);
     }
+
+    @Override
+    public boolean exists(String elementId) {
+        File file = new File(_dataFolder, elementId);
```

Comment:
can this throw an IOException in any way and make the caller of this method fail in an uncaught way?

Reply:
We're safe here, it only throws NullPointerException if some argument is null.